doc/fwutil/fwutil.md

```diff
@@ -85,18 +95,18 @@ This document provides general information about FW utility implementation in SO
 
 ## 1.1 Feature overview
 
-A modern network switch is a sophisticated equipment which consists of many auxiliary components  
-which are responsible for managing different subsystems (e.g., PSU/FAN/QSFP/EEPROM/THERMAL)  
```

Comment:
@sujinmkang why do we need to remove these whitespaces? Basically in `.md` two space notation is used to start a new line which allows text separation.